nibabel/parrec.py

```diff
@@ -609,7 +609,7 @@ def __getitem__(self, slicer):
                                  'F')
         # Broadcast scaling to shape of original data
         slopes, inters = self._slice_scaling
-        fake_data = strided_scalar(self._shape)
```

Comment:
I think you just need to create a new array, as `self._shape` is probably read-only. Perhaps 
```
        fake_data = strided_scalar(self._shape.copy())
```
 will work.